packages/suite/src/components/onboarding/OnboardingCancelButtonContext.tsx

```diff
@@ -0,0 +1,25 @@
+import { ReactNode, createContext, useContext, useState } from 'react';
+
+type CancelButtonContextData = {
+    onCancelHandler: (() => void) | null;
+    setOnCancelHandler: null | ((onCancelHandler: (() => void) | null) => void);
+};
```

Comment:
all the arrows and nulls and voids make it unclear at first glance. How about this?

```suggestion
type OnCancelHandler = () => void;

type CancelButtonContextData = {
    onCancelHandler: OnCancelHandler | null;
    setOnCancelHandler: Dispatch<OnCancelHandler> | null;
};
```

Note: Generally, you type a `useState<T>` setter function with `Dispatch<SetStateAction<T>>`, but because here `T` is _a function type_ , it is unlikely someone would want to use it with the `prevState => newState` syntax, so just `Dispatch` will do, to make it simpler :smiley:

Reply:
:+1: